### PR TITLE
Add logical router status collector

### PR DESCRIPTION
**Pull request by girikuncoro**
Logical router status API doesn't provide explicit health status of
logical router, the logical router status is bound to transport node.
So, this commit generates status from each transport node and assume 1.0
as status.

Signed-off-by: Giri Kuncoro <girikuncoro@gmail.com>

### CODE REVIEW

File: collector/logical_router_collector.go [hunk 1]
```diff
@@ -67,14 +84,40 @@ func (c *logicalRouterCollector) Collect(ch chan<- prometheus.Metric) {
 		level.Error(c.logger).Log("msg", "Unable to list logical routers", "err", err)
 		return
 	}
+	logicalRouterStatusMetrics := c.generateLogicalRouterStatusMetrics(logicalRouters)
+	for _, lrouterMetric := range logicalRouterStatusMetrics {
+		labels := []string{lrouterMetric.ID, lrouterMetric.Name, lrouterMetric.TransportNodeID, lrouterMetric.ServiceRouterID, lrouterMetric.HighAvailabilityStatus}
+		ch <- prometheus.MustNewConstMetric(c.logicalRouterStatus, prometheus.GaugeValue, 1.0, labels...)
```
Comment (walbertus) [inline collector/logical_router_collector.go]: Shouldn't we include the value of all possible status?

Comment (girikuncoro) [reply collector/logical_router_collector.go]: fixed in  142421d

**Review by walbertus — APPROVED** [review]
lgtm

**Comment by girikuncoro** [comment]
Address #57

**Pull request merged**